cmd/crc/cmd/root.go

```diff
@@ -186,3 +193,7 @@ func attachMiddleware(names []string, cmd *cobra.Command) {
 		cmd.RunE = executeWithLogging(fullCmd, src)
 	}
 }
+
+func crcManPageGenerator(targetDir string) error {
+	return doc.GenManTree(rootCmd, manpages.CrcManPageHeader, targetDir)
+}
```

Comment:
how much extra time does it take when we generate it first time?

Reply:
I'm trying to measure execution time using the `time` command however I'm not seeing any significant difference in execution times. Execution time comes out different on every execution:

In both cases I ran `crc cleanup` to make sure no man pages were generated.

Time for `crc setup` on v2.46.0:
```
real    0m4.171s
user    0m0.127s
sys     0m0.140s
```

Time for `crc setup` based on this PR:
```
real    0m3.906s
user    0m0.097s
sys     0m0.108s
```